File: 2390-removing-stars-from-a-string.py
class Solution:
    def removeStars(self, s: str) -> str:
        stack = []
        i = 0
        while i < len(s):
            c = s[i]
            if c == '*':
                stack.pop()
            else:
                stack.append(c)
            i += 1
        return "".join(stack)

File: 2390-removing-stars-from-a-string.rs
impl Solution {
    pub fn remove_stars(s: String) -> String {
        let mut res = String::new();
        for c in s.chars() {
            if c == '*' {
                res.pop();
            } else {
                res.push(c);
            }
        }
        res
    }
}
